bin/prover-client/src/prover.rs

```diff
@@ -338,22 +339,23 @@ impl Prover {
         }
     }
 
-    fn save_proof_to_db(&self, task_id: Uuid, proof: &Proof) -> Result<(), anyhow::Error> {
-        self.db
-            .prover_task_db()
-            .insert_new_task_entry(*task_id.as_bytes(), proof.into())?;
+    // TODO: replace `task_id` with ProofId
+    fn save_proof_to_db(&self, _task_id: Uuid, _proof: &Proof) -> Result<(), anyhow::Error> {
+        // Note: This works fine for now because proof is never read from DB right now
+        // let proof_id = ProofId::BtcBlockspace(1);
+        // self.db.proof_db().insert_proof(proof_id, proof.clone())?;
```

Comment:
Is this supposed to be commented out?

Reply:
Yes, for now! Since we don't use DB is the prover-client, this is fine for now.
This will be used on the upcoming PR that'll actually use this.
This is done so to reduce the scope of this PR.